sql/parser/sql.go

```diff
@@ -333,313 +333,319 @@ var yyExca = []int{
 	-2, 0,
```

Comment:
Is there a comment that causes github to omit diff? My understanding is that github omit diffs based on filename patterns (e.g. `.pb.go`).